builders/container-apps-internal-registry-demo/startup-script.sh

```diff
@@ -52,48 +52,41 @@ token=$(printf "%s" "$REGISTRY_AUTH_USERNAME:$REGISTRY_AUTH_PASSWORD" | base64)
 acr_access_string="Basic $token"
 export CNB_REGISTRY_AUTH='{"'$ACR_RESOURCE_NAME'":"'$acr_access_string'"}'
 
+echo "Initiating buildpack build..."
+echo "Correlation id: '$CORRELATION_ID'"
+
```

Comment:
nit: it may be worth either putting an extra `echo` here or putting a `\n` at the end of the correlation ID `echo` so that there's separation between this message and the "Executing the analyze phase" message below